test/test.config.js

```diff
@@ -0,0 +1,13 @@
+const { Pool } = require('pg');
+
+const testDatabaseConfig = {
+  user: 'testuser',
+  host: 'localhost',
+  database: 'testdb',
+  password: 'testpassword',
+  port: 5432,
+};
+
+const testPool = new Pool(testDatabaseConfig);
+
+module.exports = { testPool };
```

Comment:
What is this for?

Reply:
Oh maybe this is read by mocha?

Reply:
Yes, you're correct. Mocha reads this configuration when executing integration tests that require interaction with the PostgreSQL database. This separate test configuration allows us to maintain a clean testing environment, ensuring that our tests do not affect production or development data, vice versa.